### PR TITLE
Contact us

### LINKED ISSUE #50: contact page

### PR DESCRIPTION
**Pull request by basil-kawak**
PTAL

### CODE REVIEW

File: src/components/contact/sections/ContactUs.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                  <i className="fa fa-facebook"></i>
+                </Nav.Link>
+                <Nav.Link href="#" className="iconTwitter" title="Facebook">
+                  <i className="fa fa-twitter"></i>
+                </Nav.Link>
+              </Nav>
+            </Col>
+          </div>
+        </div>
+        <div className="formSide">
```
Comment (gbilgedal) [inline src/components/contact/sections/ContactUs.jsx:40]: maybe an empty line before this would be great

**Review by gbilgedal — APPROVED** [review]
looks pretty clean to me, there is a typo on one of the commits tho :D
and also, what about translation?

File: src/components/contact/sections/ContactUs.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+import { Container, Col, Form, Button, Nav } from "react-bootstrap";
+
+const ContactUs = () => {
+  const initContactFormState = {
+    name: "",
+    surName: "",
+    email: "",
+    message: "",
+  };
+  const [form, setform] = useState(initContactFormState);
```
Comment (louisrli) [inline src/components/contact/sections/ContactUs.jsx]: setform should be setForm

File: src/components/contact/sections/ContactUs.jsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import { Container, Col, Form, Button, Nav } from "react-bootstrap";
+
+const ContactUs = () => {
+  const initContactFormState = {
+    name: "",
+    surName: "",
```
Comment (louisrli) [inline src/components/contact/sections/ContactUs.jsx]: surName should just be surname as it is one word in english

however i would prefer line 6 be firstName and line 7 be lastName (more common than name/surname)

File: src/components/contact/sections/ContactUs.jsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                </Nav.Link>
+                <Nav.Link href="#" className="iconTwitter" title="Facebook">
+                  <i className="fa fa-twitter"></i>
+                </Nav.Link>
+              </Nav>
+            </Col>
+          </div>
+        </div>
+        <div className="formSide">
+          <h3>CONTACT US</h3>
```
Comment (louisrli) [inline src/components/contact/sections/ContactUs.jsx]: same thing, use text-transform here and just type "Contact Us" on line 41

**Review by louisrli — APPROVED** [review]
generally lgtm please fix the comments beofre merging

**Comment by louisrli** [comment]
please write snapshot tests i forgot

**Pull request merged**